### PR TITLE
[5.5] Add str_words() helper function

### PR DESCRIPTION
**Pull request by paulofreitas**
It's been there for a long time and was never promoted to a helper function although it's useful enough. 😄

### CODE REVIEW

File: tests/Support/SupportHelpersTest.php [hunk 1]
```diff
@@ -315,6 +315,21 @@ public function testStrLimit()
...
+        $this->assertEquals('Hello...', str_words($string, 1));
+        $this->assertEquals('Hello___', str_words($string, 1, '___'));
+        $this->assertEquals('Hello', str_words($string, 1, ''));
+        $this->assertEquals('Hello world!', str_words($string, 3));
+
+        $unicodeString = 'Привет мир!';
+        $this->assertEquals('Привет...', str_words($unicodeString, 1));
+        $this->assertEquals('Привет___', str_words($unicodeString, 1, '___'));
+        $this->assertEquals('Привет', str_words($unicodeString, 1, ''));
+        $this->assertEquals('Привет мир!', str_words($unicodeString, 3));
```
Comment (stefanoruth) [inline tests/Support/SupportHelpersTest.php]: As you can see i in the other Str tests, its the classs that is used not the helper :)

Comment (paulofreitas) [reply tests/Support/SupportHelpersTest.php]: @stefanoruth The `Str` tests belongs to `SupportStrTest` class. Yup, things got mixed somehow.

**Comment by taylorotwell** [comment]
I really would rather move towards getting rid of the string and array helper functions and preferring the class when possible.

**Comment by ntzm** [comment]
@taylorotwell Maybe that could be written in the contributing file? So you don't get spammed with these and people don't waste their time.

**Pull request closed without merging**